### PR TITLE
refactor(experimental): rename accounts errors

### PR DESCRIPTION
**Pull request by lorisleiva**
Renames the following accounts errors:

```ts
// Before.
export const SOLANA_ERROR__ACCOUNT_NOT_FOUND = 3230001 as const;
export const SOLANA_ERROR__MULTIPLE_ACCOUNTS_NOT_FOUND = 32300002 as const;
export const SOLANA_ERROR__FAILED_TO_DECODE_ACCOUNT = 3230003 as const;
export const SOLANA_ERROR__EXPECTED_DECODED_ACCOUNT = 3230004 as const;
export const SOLANA_ERROR__NOT_ALL_ACCOUNTS_DECODED = 3230005 as const;

// After.
export const SOLANA_ERROR__ACCOUNTS_ACCOUNT_NOT_FOUND = 3230001 as const;
export const SOLANA_ERROR__ACCOUNTS_ONE_OR_MORE_ACCOUNTS_NOT_FOUND = 32300002 as const;
export const SOLANA_ERROR__ACCOUNTS_FAILED_TO_DECODE_ACCOUNT = 3230003 as const;
export const SOLANA_ERROR__ACCOUNTS_EXPECTED_DECODED_ACCOUNT = 3230004 as const;
export const SOLANA_ERROR__ACCOUNTS_EXPECTED_ALL_ACCOUNTS_TO_BE_DECODED = 3230005 as const;
```

### CODE REVIEW

**Comment by lorisleiva** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2259?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2262** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2262?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2261** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2261?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2260** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2260?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2259** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2259?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2257** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2257?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2256** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2256?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2255** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2255?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2254** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2254?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2253** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2253?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2252** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2252?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2251** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2251?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2250** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2250?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2249** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2249?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lorisleiva and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by buffalojoec — APPROVED** [review]
Lgtm just starting to see some import linter errors!

File: packages/errors/src/messages.ts [hunk 1]
```diff
@@ -196,7 +196,12 @@ export const SolanaErrorMessages: Readonly<{
     // TypeScript will fail to build this project if add an error code without a message.
     [P in SolanaErrorCode]: string;
 }> = {
-    [SOLANA_ERROR__ACCOUNT_NOT_FOUND]: 'Account not found at address: $address',
+    [SOLANA_ERROR__ACCOUNTS_ACCOUNT_NOT_FOUND]: 'Account not found at address: $address',
+    [SOLANA_ERROR__ACCOUNTS_EXPECTED_ALL_ACCOUNTS_TO_BE_DECODED]:
+        'Not all accounts were decoded. Encoded accounts found at addresses: $addresses.',
```
Comment (steveluscher) [inline packages/errors/src/messages.ts:201]: But what's your opinion of error messages that end in periods? :)

**Review by steveluscher — APPROVED** [review]
Added via Giphy

File: packages/errors/src/messages.ts (hunk 1 above)
Comment (lorisleiva) [reply packages/errors/src/messages.ts:201]: I don't mind either way but it should be all or nothing IMO. 🫡

**Pull request merged**